### PR TITLE
feat(style-panel): modify this style-panel styles according to the design craft

### PR DESCRIPTION
**Pull request by betterdancing**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Release Notes

- **New Features**
	- Introduced a new `TabsGroupConfigurator` component for improved tabbed interfaces across various settings.
	- Enhanced `NumberConfigurator` with unit selection and improved layout.
	- Added support for selecting background types and sizes using the new `tabs-group-configurator`.
	- New prop `teleport` in `ModalMask.vue` for customizable modal positioning.
	- Added new CSS variables for enhanced theming options.

- **Improvements**
	- Streamlined user interactions in several components, including `ImageSetting` and `RadialGradient`, by replacing tooltips with tabbed selections.
	- Updated styling across components for a more cohesive look and feel, including adjustments to dimensions and colors.
	- Enhanced layout and visual presentation in the `BackgroundGroup` and `TypographyGroup` components.
	- Refined typography settings interface for better organization and user interaction.
	- Improved the layout options in `LayoutGroup` for clarity and usability.
	- Enhanced visual representation of spacing controls in `SpacingGroup`.
	- Adjusted CSS variables for better theming and consistency across styles.

- **Bug Fixes**
	- Fixed input handling in `NumericSelect` to ensure only valid numeric entries are accepted.

- **Chores**
	- Updated internal constants for consistency in label casing and terminology throughout the application.

### CODE REVIEW

File: packages/configurator/src/slider-configurator/SliderConfigurator.vue [hunk 1]
```diff
@@ -133,7 +133,7 @@ export default {
     padding: 0;
     cursor: pointer;
     border: 0;
-    background: -webkit-linear-gradient(#fff, #fff) no-repeat #999999;
+    background: -webkit-linear-gradient(#fff, #fff) no-repeat #ccc;
```
Comment (chilingling) [inline packages/configurator/src/slider-configurator/SliderConfigurator.vue]: ccc 是色板里面的嘛？可以用样式变量嘛？

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+          @click.stop="change(foldsItem.value)"
+        >
+          <span v-if="foldsItem.label">{{ foldsItem.label }}</span>
+          <svg-icon v-if="foldsItem.icon" :name="foldsItem.icon" class="bem-Svg"></svg-icon>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 新组件建议用 setup 的写法  `<script setup>`

File: packages/settings/styles/src/Main.vue [hunk 3]
```diff
@@ -246,4 +260,12 @@ export default {
     }
   }
 }
+
+.dots {
+  display: inline-block;
+  margin-left: 4px;
+  vertical-align: middle;
+  border: 2px solid #1476ff;
```
Comment (chilingling) [inline packages/settings/styles/src/Main.vue]: 为什么不用 css 变量？

File: packages/configurator/src/slider-configurator/SliderConfigurator.vue [hunk 4]
```diff
@@ -145,7 +146,7 @@ export default {
     height: 10px;
     border-radius: 50%;
     border: 0;
-    background-color: #fff;
+    background-color: var(--ti-lowcode-base-bg-5);
```
Comment (wenmine) [inline packages/configurator/src/slider-configurator/SliderConfigurator.vue]: 还是要调整一下，base不是用的这个，应该用的--te-base开头的

File: packages/configurator/src/slider-configurator/SliderConfigurator.vue [hunk 5]
```diff
@@ -133,7 +133,8 @@ export default {
     padding: 0;
     cursor: pointer;
     border: 0;
-    background: -webkit-linear-gradient(#fff, #fff) no-repeat #ccc;
+    background: -webkit-linear-gradient(var(--ti-lowcode-base-gray-0), var(--ti-lowcode-base-gray-0)) no-repeat
+      var(--ti-lowcode-breadcrumb-icon-color);
```
Comment (chilingling) [inline packages/configurator/src/slider-configurator/SliderConfigurator.vue]: 变量这里乱用了，检查一下变量的语义，不要乱用

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 6]
```diff
@@ -8,9 +8,9 @@
       @click.stop="change(item.value)"
     >
       <span :class="['label-text', index < commonOptions.length - 1 ? 'border-right' : '']">
-        <span v-if="item?.label">{{ item.label }}</span>
+        <span v-if="item && item.label">{{ item.label }}</span>
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 原来 optional chaining 的用法就挺好的，AI 的 review 不一定都要采纳，自己甄别一下

`item?.label`

File: packages/settings/styles/src/components/background/BackgroundGroup.vue [hunk 7]
```diff
@@ -418,18 +429,17 @@ export default {
     place-items: stretch;
     gap: 1px;
     border-radius: 2px;
-    border-width: 1px;
-    border-style: solid;
-    border-color: #2b2b2b;
-    background-color: #2b2b2b;
-    margin-bottom: 12px;
+    border-top: 1px solid var(--te-base-invalidate-color);
```
Comment (chilingling) [inline packages/settings/styles/src/components/background/BackgroundGroup.vue]: 这里变量看看是不是不符合语义化

File: packages/settings/styles/src/components/background/BackgroundGroup.vue [hunk 8]
```diff
@@ -418,18 +429,17 @@ export default {
...
+    margin: 12px 0;
     .image-list-item {
       display: grid;
       grid-template-columns: 8px 16px minmax(auto, 1fr) auto;
       gap: 8px 4px;
       height: 24px;
       align-items: center;
-      background-color: rgb(54, 54, 54);
+      background-color: var(--te-base-bg-5);
+      border-bottom: 1px solid var(--te-base-invalidate-color);
```
Comment (chilingling) [inline packages/settings/styles/src/components/background/BackgroundGroup.vue]: 这里变量看看是不是不符合语义化

File: packages/theme/light/setting-style.less [hunk 9]
```diff
@@ -5,4 +5,10 @@
   --ti-lowcode-setting-style-btn-font-color: var(--ti-lowcode-base-gray-50);
   --ti-lowcode-setting-style-btn-border-color: var(--ti-lowcode-base-gray-90);
   --ti-lowcode-setting-style-title-color: var(--ti-lowcode-base-text-color);
+  --ti-lowcode-setting-style-tab-bg-color: var(--ti-lowcode-base-bg-7);
+  --ti-lowcode-setting-style-reset-bg-color: var(--ti-lowcode-base-text-color-2);
+  --ti-lowcode-setting-style-reset-dot-color: var(--ti-lowcode-base-text-color-2);
+  --ti-lowcode-setting-style-bg-box-shadow: #212121;
+  --ti-lowcode-setting-style-drag-bar-bg: #8ac2ff;
```
Comment (chilingling) [inline packages/theme/light/setting-style.less]: 色板里没有这个色值，跟UI确认一下

File: packages/settings/styles/src/components/inputs/NumericSelect.vue [hunk 10]
```diff
@@ -6,9 +6,20 @@
       @focus="focus = true"
       @blur="focus = false"
       @change="change"
+      @input="input"
     >
       <template #suffix>
-        <span class="suffix">{{ numericalSuffix }}</span>
+        <div class="suffix-wrap">
+          <div class="add-substact-wrap">
```
Comment (chilingling) [inline packages/settings/styles/src/components/inputs/NumericSelect.vue]: image
上下加减按钮的空间不需要自己实现，组件库有。
https://opentiny.design/tiny-vue/zh-CN/smb-theme/components/numeric#controls

File: packages/settings/styles/src/components/inputs/ModalMask.vue [hunk 11]
```diff
@@ -38,9 +42,33 @@ export const useModal = () => {
...
 export default {
-  setup() {
+  props: {
+    teleport: {
+      type: String,
+      default: '.tiny-engine-right-wrap'
+    }
+  },
+  setup(props) {
+    const isAlignBody = ref(props.teleport === 'body')
```
Comment (chilingling) [inline packages/settings/styles/src/components/inputs/ModalMask.vue]: 这个不会变的吧？不需要用 ref。直接 `const isAlignBody = props.teleport === 'body'` 就行了？

File: packages/settings/styles/src/components/inputs/ModalMask.vue [hunk 12]
```diff
@@ -1,17 +1,21 @@
...
-  <teleport to=".tiny-engine-right-wrap">
+  <teleport :to="teleport">
     <div class="modal-wrapper">
-      <div class="modal-mask" @click="$emit('close')"></div>
+      <div :class="[isAlignBody ? '' : 'modal-mask']" @click="$emit('close')"></div>
 
-      <div :style="{ top: modal.top - 30 + 'px' }" class="modal-content">
+      <div
+        ref="modalContent"
+        :style="{ top: topStyle + 'px' }"
```
Comment (chilingling) [inline packages/settings/styles/src/components/inputs/ModalMask.vue]: 建议用模板字符串，`${topStyle}px`

File: packages/configurator/src/color-configurator/ColorConfigurator.vue [hunk 13]
```diff
@@ -23,7 +23,7 @@ export default {
     },
     modelValue: {
       type: String,
-      default: ''
+      default: '#FFFFFF'
```
Comment (chilingling) [inline packages/configurator/src/color-configurator/ColorConfigurator.vue]: 这里为什么要默认为 `#FFFFFF` ?

File: packages/configurator/src/color-configurator/ColorConfigurator.vue [hunk 14]
```diff
@@ -68,6 +68,10 @@ export default {
   :deep(.tiny-input-prefix) {
     .tiny-input__prefix {
       left: 2px;
+
+      input[type='color' i] {
```
Comment (chilingling) [inline packages/configurator/src/color-configurator/ColorConfigurator.vue]: `input[type=`color` i]` 是什么语法？没见过

File: packages/configurator/src/slider-configurator/SliderConfigurator.vue [hunk 15]
```diff
@@ -133,20 +133,21 @@ export default {
...
   }
+
   input[type='range']::-webkit-slider-thumb {
     width: 10px;
     height: 10px;
     border-radius: 50%;
     border: 0;
     background-color: var(--te-common-bg-default);
-    box-shadow: 0 1px 4px rgba(0, 0, 0, 0.21);
+    box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.4);
```
Comment (chilingling) [inline packages/configurator/src/slider-configurator/SliderConfigurator.vue]: 基础色板 base 有对应变量嘛？能不能用基础色板变量替换？

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 16]
```diff
@@ -270,6 +271,7 @@ const change = (item) => {
   margin: 0px 0px 0px 20px;
   background-color: rgb(255, 255, 255);
   z-index: 9999;
+  box-shadow: 0 0 10px 0 rgba(25, 25, 25, 0.15);
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 看看有没有对应的阴影色值变量

File: packages/common/component/MetaCodeEditor.vue [hunk 17]
```diff
@@ -298,7 +298,7 @@ export default {
   justify-content: space-between;
   align-items: center;
   width: 100%;
-  height: 32px;
+  height: 24px;
```
Comment (chilingling) [inline packages/common/component/MetaCodeEditor.vue:303]: image
样式有问题了，溢出了

File: packages/settings/styles/src/components/size/SizeGroup.vue [hunk 18]
```diff
@@ -33,7 +33,7 @@
         :class="['size-label', { 'is-setting': getSettingFlag(SIZE_PROPERTY.MinWidth) }]"
         @click="openSetting(SIZE_PROPERTY.MinWidth, $event)"
       >
-        最小宽
+        <span>最小宽</span>
```
Comment (chilingling) [inline packages/settings/styles/src/components/size/SizeGroup.vue:36]: image

最大宽最小宽重置之后需要显示为空，不能显示为0。空值和 0 值是不一样的。
其他属性也检查一下。

File: packages/theme/light/setting-style.less [hunk 19]
```diff
@@ -5,4 +5,9 @@
   --ti-lowcode-setting-style-btn-font-color: var(--ti-lowcode-base-gray-50);
   --ti-lowcode-setting-style-btn-border-color: var(--ti-lowcode-base-gray-90);
   --ti-lowcode-setting-style-title-color: var(--ti-lowcode-base-text-color);
+  --ti-lowcode-setting-style-tab-bg-color: var(--ti-lowcode-base-bg-7);
+  --ti-lowcode-setting-style-reset-bg-color: var(--ti-lowcode-base-text-color-2);
```
Comment (chilingling) [inline packages/theme/light/setting-style.less]: bg colo 为什么要用基础的 text-color? 貌似不符合语义化？

File: packages/design-core/assets/style-panel-expand.svg [hunk 20]
```diff
@@ -0,0 +1,11 @@
+<svg width="16.000000" height="16.000000" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<desc>
+			Created with Pixso.
+	</desc>
```
Comment (hexqi) [inline packages/design-core/assets/style-panel-expand.svg]: desc和defs可以去掉

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 21]
```diff
@@ -0,0 +1,300 @@
...
+  },
+  valueKey: {
+    type: String,
+    default: ''
+  },
+  effect: {
+    type: String,
+    default: 'dark'
+  },
+  placement: {
```
Comment (hexqi) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue:114]: props需要补充注释，这是一个基础公共组件，后面很可能被其他场景和外部用户用到

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 22]
```diff
@@ -0,0 +1,300 @@
...
+const props = defineProps({
+  modelValue: {
+    type: String,
+    default: ''
+  },
+  valueKey: {
+    type: String,
+    default: ''
+  },
+  effect: {
```
Comment (hexqi) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: effect这个属性没有必要，tiny-popover文档中已经没有effect这个属性，并且显示效果应该由tinyvue全局主题切换控制

File: packages/layout/src/DesignSettings.vue [hunk 23]
```diff
@@ -2,10 +2,15 @@
   <div id="tiny-right-panel">
     <tiny-tabs v-model="layoutState.settings.render">
       <tiny-tab-item v-for="(setting, index) in settings" :key="index" :title="setting.title" :name="setting.name">
-        <component :is="setting.entry"></component>
+        <component :is="setting.entry" :is-collapsed="isCollapsed"></component>
         <div v-show="activating" class="active"></div>
       </tiny-tab-item>
     </tiny-tabs>
+    <div class="tabs-setting">
+      <tiny-tooltip effect="dark" :content="isCollapsed ? '展开' : '折叠'" placement="top" :visible-arrow="false">
```
Comment (hexqi) [inline packages/layout/src/DesignSettings.vue:10]: 这个icon有什么作用？点击后就隐藏了，没有任何效果
image

File: packages/settings/styles/src/components/spacing/SpacingGroup.vue [hunk 24]
```diff
@@ -1,18 +1,23 @@
 <template>
   <div class="spacing-wrap">
     <div class="spacing-max-icon">
-      <svg xmlns="http://www.w3.org/2000/svg" width="224" height="120" style="grid-area: 1 / 1 / -1 / -1">
+      <svg
```
Comment (hexqi) [inline packages/settings/styles/src/components/spacing/SpacingGroup.vue:4]: 提取到单独的svg文件里面吧

File: packages/theme/light/setting-style.less [hunk 25]
```diff
@@ -5,4 +5,9 @@
   --ti-lowcode-setting-style-btn-font-color: var(--ti-lowcode-base-gray-50);
   --ti-lowcode-setting-style-btn-border-color: var(--ti-lowcode-base-gray-90);
   --ti-lowcode-setting-style-title-color: var(--ti-lowcode-base-text-color);
+  --ti-lowcode-setting-style-tab-bg-color: var(--ti-lowcode-base-bg-7);
+  --ti-lowcode-setting-style-reset-bg-color: var(--te-common-bg-primary-checked);
```
Comment (hexqi) [inline packages/theme/light/setting-style.less]: 这个变量没有看到哪里有用到，可以删掉

File: packages/theme/light/setting-style.less [hunk 26]
```diff
@@ -5,4 +5,9 @@
...
+  --ti-lowcode-setting-style-reset-dot-color: var(--te-common-bg-primary-checked);
```
Comment (hexqi) [inline packages/theme/light/setting-style.less]: 建议直接引用符合场景的te变量，不要随意重新新增ti变量

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 27]
```diff
@@ -0,0 +1,300 @@
...
+  }
+}
+.tiny-dropdown-menu {
+  padding: 8px 0px;
+  margin: 0px 0px 0px 20px;
+  background-color: var(--te-common-bg-default);
+  color: var(--te-common-text-weaken);
+  z-index: 9999;
+  box-shadow: 0 0 10px 0 var(--te-common-border-default);
+  --ti-dropdown-menu-arrow-margin-top: 0;
```
Comment (hexqi) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue:277]: 通过设置样式值方式覆盖样式，不要直接覆盖cssvar变量，如果需要全局覆盖，定义到base主题里面

File: packages/settings/styles/src/components/spacing/SpacingGroup.vue (hunk 24 above)
Comment (betterdancing) [reply packages/settings/styles/src/components/spacing/SpacingGroup.vue:4]: 由于svg都写有样式，迁移起来改动会比较多，建议放到单独的PR里提交

**Review by hexqi — APPROVED** [review]

**Review by rhlin — APPROVED** [review]

**Pull request merged**